source/css/scss/app/pages/finding.scss

```diff
@@ -2,16 +2,23 @@
 @import '../../button-vars';
 @import '../../font-vars';
 
-$filter-menu-base-width: rem-calc(251);
-$flexbox-gutter-up-breakpoint: "#{$screen} and (min-width: 1020px)";
-$flexbox-gutter-only-breakpoint: "#{$screen} and (min-width: 1020px) and (max-width: 1529px)";
-$filters-open-card-count-breakpoint: "#{$screen} and (min-width: 1530px)";
+$filter-menu-px-width: 251px;
+$filter-menu-base-width: rem-calc($filter-menu-px-width);
```

Comment:
does rem-calc work OK with a value expressed in px? (same question on line 9)

Reply:
Yes it does: https://github.com/zurb/foundation-sites/blob/v5.5.3/scss/foundation/_functions.scss#L109
The calculation actually strips the units off whatever input you give it so defining as px is fine.